### DISCUSSION TITLE
Is there a plan to exit the program without saving the file？

### DISCUSSION THREAD

**Discussion by itInner** [dail8859/NotepadNext#295] [Ideas]
put the file in the internal cache file of the program，just like n++
The next time you open the program, the file is opened but not closed by default 
Low code scenario, temporary saving of text, but do not want to save file to disk

**Comment by dail8859** [comment]
Pull Request #264 was just recently merged after a few months' work on this feature. It will be available in the next release.

If you are interested in testing it out and providing feedback you can find links to download test builds which include this feature here: https://github.com/dail8859/NotepadNext/issues/81#issuecomment-1367378181
